src/tests/test_magic_cmd.py

```diff
@@ -585,3 +643,35 @@ def test_delete_invalid_snippet(arg, ip_snippets):
         str(out.error_in_exec) == "No such saved snippet found "
         ": non_existent_snippet"
     )
+
+
+def test_table_explore_with_snippets(ip, tmp_empty):
+    ip.run_cell(
+        """
+        %%sql duckdb://
+        CREATE TABLE people (name varchar(50), number int, country varchar(50));
+        INSERT INTO people VALUES ('joe', 82, 'usa');
+        INSERT INTO people VALUES ('paula', 93, 'uk');
+        INSERT INTO people VALUES ('sam', 77, 'canada');
+        INSERT INTO people VALUES ('emily', 65, 'usa');
+        INSERT INTO people VALUES ('michael', 89, 'usa');
+        INSERT INTO people VALUES ('sarah', 81, 'uk');
+        INSERT INTO people VALUES ('james', 76, 'usa');
+        INSERT INTO people VALUES ('angela', 88, 'usa');
+        INSERT INTO people VALUES ('robert', 82, 'usa');
+        INSERT INTO people VALUES ('lisa', 92, 'uk');
+        INSERT INTO people VALUES ('mark', 77, 'usa');
+        INSERT INTO people VALUES ('jennifer', 68, 'australia');
+        """
+    )
+    ip.run_cell(
+        """
+        %%sql --save citizen
+        select * from people where country = 'usa'
+        """
+    )
+    with capture_output() as captured:
+        ip.run_cell("%sqlcmd explore -t citizen")
+
+    assert "Plotting using saved snippet : citizen" in captured.stdout
```

Comment:
I meant we can remove `Plotting` and add something more generic. I think this message is getting invoked for all of plot/ explore/ profile? is there a way to find which function it got called from and change the message accordingly?  Like if called from `explore` it should display `Exploring using saved snippet : <snippet>` . If this is not possible we can display something more generic like `analysing using saved snippet` maybe.
What do you think @edublancas

Reply:
Made the change, now we will display a message based on the command that has called the `is_saved_snippet_or_table_exists` .
eg: Profiling using the saved snippet
      Exploring using the saved snippet
      Plotting using the saved snippet
      default : Analyzing using the saved snippet


<img width="619" alt="Screenshot 2023-07-13 at 1 40 24 PM" src="https://github.com/ploomber/jupysql/assets/69951190/50c28016-60d3-4e3d-b864-fdebae5c1ecf">